### PR TITLE
Add a template for creating new issues

### PR DESCRIPTION
**Pull request by patcon**
Just wanted to create a quick one. We can have a few, and this doesn't need to cover everything (a blank is always still an option).

Even if we create through ZenHub most of the time (and bypass GitHub's templates), this can be a file to copy-paste from for an idealized format.

Screen Shot 2022-05-03 at 6 11 41 PM

### CODE REVIEW

File: .github/issue_template.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+about: Use this template new features with links to prototypes.
+# These aren't used at the moment.
+# See: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository
+title: null
+labels: null
+assignees: null
+---
+<sup>ℹ️ [Figma (root)][figma] | [Sitemap][sitemap]</sup>
+<sup>_This initial comment is collaborative and open to modification by all._</sup>
+
```
Comment (tristan-orourke) [inline .github/issue_template.yml:12]: Maybe add a `## Description` section to delineate from the `<sup>` section?

**Review by tristan-orourke — COMMENTED** [review]
Yeah, this will be handy! I just left a couple suggestions. And the template can be used the same way in ZenHub, by the way.

**Review by tristan-orourke — CHANGES_REQUESTED** [review]
@patcon I left a couple comments before, but maybe you didn't see them because I forgot to make it a official review

File: .github/issue_template.yml (hunk 1 above)
Comment (patcon) [reply .github/issue_template.yml:12]:
```suggestion

## Description
```

**Pull request merged**